frontend/pulse3d/src/utils/labwareCalculations.js

```diff
@@ -100,4 +100,17 @@ export class WellTitle {
   getWellIndexFromRowColumn(rowIndex, columnIndex) {
     return columnIndex * this.numRows + rowIndex;
   }
+  /**
+   * Get the well index from well name
+   *
+   * @param {string} wellName
+   * @returns {int}
+   */
+  wellNameToIndex(wellName) {
+    const row = wellName.charCodeAt(0) - "A".charCodeAt(0);
+    const col = parseInt(wellName.slice(1)) - 1;
+    const numCols = 6;
```

Comment:
num_cols is already defined in the constructor on line 27 https://github.com/CuriBio/k8s_curi/pull/486/files#diff-f070e8d7209e1b23b3985652cefa077c2a575132a1cd7ebf420d3183f84b5bbaR27. We should use that value instead of redefining. Also, we should make that value set in the constructor `numCols` instead of `num_cols`